firewood/src/merkle/node.rs

```diff
@@ -243,15 +243,19 @@ impl LeafNode {
 }
 
 #[derive(PartialEq, Eq, Clone)]
-pub struct ExtNode(
-    pub(super) PartialPath,
-    pub(super) DiskAddress,
-    pub(super) Option<Vec<u8>>,
-);
+pub struct ExtNode {
+    path: PartialPath,
+    chd: DiskAddress,
+    chd_encoded: Option<Vec<u8>>,
```

Comment:
nit:
```suggestion
    child: DiskAddress,
    child_encoded: Option<Vec<u8>>,
```

Reply:
After reading through all of the code here, I definitely think it's an improvement, but we really shouldn't be using a getter pattern with what is effectively an internal type like this. I don't think it provides any benefit. I would make all of these fields `pub(super)` and the struct itself `pub(crate)` assuming that the node-structure never actually leaks outside of the database.